doc/en/user/source/community/hana/index.rst

```diff
@@ -0,0 +1,86 @@
+.. _community_hana:
+
+SAP HANA
+========
+
+Supported versions
+------------------
+
+The module supports HANA 1 SPS 12 and all HANA 2 versions including the `free express edition <https://www.sap.com/sap-hana-express>`_.
+
+.. _hana_install:
+
+Installing SAP HANA support
+---------------------------
+
+GeoServer has no built-in support for SAP HANA. You can enable SAP HANA support by adding the HANA GeoTools module and the HANA JDBC driver to your GeoServer installation.
+
+Installing the HANA GeoTools module
+```````````````````````````````````
+
+Identify the GeoTools version your GeoServer installation is using by navigating to the :file:`webapps/geoserver/WEB-INF/lib/` folder in your GeoServer installation and locating the file :file:`gt-jdbc-{<version>}.jar` as shown in the image below. In the example below the GeoServer version is ``22.0``.
+
+.. figure:: images/gt-version.png
+   :align: center
+
+   *Finding the GeoTools version*
```

Comment:
No need to use italics, we control appearance with the sphinx theme for captions.

Reply:
Oh wait that is what https://docs.geoserver.org/latest/en/docguide/sphinx.html has as an example, sigh :)